src/components/Display.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './Display.scss';
+import PropTypes from 'prop-types';
+
+function Display({ result }) {
+  return <h1>{result}</h1>;
+}
+
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
+
```

Comment:
just as you put the `propTypes` to string, also enforce the `defaultProps` to '0' as requirements says 
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png